### PR TITLE
preview env and cluster

### PR DESCRIPTION
**Pull request by Albanetonnellier**
Create a stop cluster page + updated stop cluster for cloud provider + update preview environment and create page environment

### CODE REVIEW

File: website/docs/using-qovery/configuration.md [hunk 1]
```diff
@@ -18,8 +18,10 @@ In the following subsections, you'll learn all you need to know to configure and
 <Jump to="/docs/using-qovery/configuration/environment/">Environment</Jump>
 <Jump to="/docs/using-qovery/configuration/object-storage/">Object storage</Jump>
 <Jump to="/docs/using-qovery/configuration/organization/">Organization</Jump>
+<Jump to="/docs/using-qovery/configuration/preview-env/">Preview env</Jump>
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration.md:21]: Please use `preview environment` and not `preview env` for the URL and the naming - stay consistent in the naming.

File: website/docs/using-qovery/configuration.md [hunk 2]
```diff
@@ -18,8 +18,10 @@ In the following subsections, you'll learn all you need to know to configure and
...
 <Jump to="/docs/using-qovery/configuration/project/">Project</Jump>
 <Jump to="/docs/using-qovery/configuration/secret/">Secret</Jump>
+<Jump to="/docs/using-qovery/configuration/stop-cluster/">Stop cluster</Jump>
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration.md:24]: why a new page for stopping the cluster? It should be on the cluster page. Not on a dedicated page.

File: website/docs/using-qovery/configuration/environment.md [hunk 3]
```diff
@@ -67,6 +67,8 @@ Sometimes **Preview Environment** are also known as **Ephemeral Environment**, *
 
 Creating a **Preview Environment** is similar to create a casual Environment.
 
+Refer to the [Preview Environment][docs.using-qovery.configuration.preview-env] documentation to know more.
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/environment.md:70]: why use a dedicated page for the preview environment? It is an Environment feature. It should not be in "configuration", but in a sub section of "Environment"

File: website/docs/using-qovery/configuration/environment.md [hunk 4]
```diff
@@ -203,5 +205,6 @@ To delete your environment, you must go in the `settings` > `general` and delete
 [docs.using-qovery.configuration.deployment-rule]: /docs/using-qovery/configuration/deployment-rule/
 [docs.using-qovery.configuration.environment]: /docs/using-qovery/configuration/environment/
 [docs.using-qovery.configuration.organization#roles]: /docs/using-qovery/configuration/organization/#roles
+[docs.using-qovery.configuration.preview-env]: /docs/using-qovery/configuration/preview-env/
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/environment.md:208]: same

File: website/docs/using-qovery/configuration/environment.md.erb [hunk 5]
```diff
@@ -58,6 +58,8 @@ Sometimes **Preview Environment** are also known as **Ephemeral Environment**, *
 
 Creating a **Preview Environment** is similar to create a casual Environment.
 
+Refer to the [Preview Environment][docs.using-qovery.configuration.preview-env] documentation to know more.
```
Comment (evoxmusic) [inline website/docs/using-qovery/configuration/environment.md.erb:61]: same

**Review by evoxmusic — CHANGES_REQUESTED** [review]

File: website/docs/using-qovery/configuration.md (hunk 1 above)
Comment (Albanetonnellier) [reply website/docs/using-qovery/configuration.md:21]: I used preview environment at the beginning but it created a conflict with the file environment and couldn't generate the md that's why I changed it, is there a way not to create a conflict?

**Comment by evoxmusic** [comment]
@Albanetonnellier can you apply my changes/suggestions?

**Comment by Albanetonnellier** [comment]
@evoxmusic this pr is no longer needed, Florian made a new page with the new cluster flow

**Pull request closed without merging**